apps/native/app/src/screens/finance/components/finance-status-card.tsx

```diff
@@ -0,0 +1,380 @@
+import {
```

Comment:
I changed the name for this from finance-status-card-_container_ to finance-status-card. I only changed styling in it, no logic.